pkg/errors/errors.go

```diff
@@ -99,7 +97,7 @@ func IsResourceVersion(err error) bool {
 type podNamespaceErr struct{}
 
 func (err *podNamespaceErr) Error() string {
-	return fmt.Sprintf("%s environment variable is not set", envutils.PodNamespaceEnvName)
+	return fmt.Sprintf("%s environment variable is not set", "POD_NAMESPACE")
```

Comment:
`statusutils.PodNamespaceEnvName` here?

Reply:
I had tried this initially, but it was causing a circular dependency, so I opted to just re-define it as a string

Reply:
Ah, gotcha.